samples/tutorial/BindingToCommandHandlers.md

```diff
@@ -5,35 +5,40 @@
 
 The simplest way to bind command line input is to set the `Command.Handler` property. The `System.CommandLine` model binder will look at the options and arguments for the command and attempt to match them to the parameters of the specified handler method. The default convention is that parameters are matched by name, so in the following example, option `--an-int` matches the parameter named `anInt`. Matching ignores hyphens (and other option prefixes, such as `'/'`) and is case insensitive.
 
-```cs --source-file ./src/Binding/HandlerBindingSample.cs --project ./src/Binding/Binding.csproj --region MultipleArgs 
+```cs
```

Comment:
Shouldn't this be `shell` not `cs`? I am not even sure `cs` as a language keyword is recognized by github.

Reply:
I guess it should be [csharp](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L542)

Reply:
Saw, your previous comment; I agree with `console`/`shell` (they are different in the yml, but I dont know what the difference is).

Reply:
This is a Try .NET code block so it's C# code. The intention is that the source code can be filled in and will display correctly here, but when run using `dotnet-try`, the code can be edited and run in the browser. In the future, filling in the code from the specified source location will be automatable (https://github.com/dotnet/try/issues/214).

I believe `cs` is acceptable. Anyhow, experimenting with it seems to produce the same coloring as `csharp`, but if `csharp` is more correct or common we can change that.

Reply:
So this would ideally be as follows in order to be both functionally correct for Try .NET while also looking correct in GitHub's Markdown file viewer:

````
```csharp  --source-file ./src/Binding/HandlerBindingSample.cs --project ./src/Binding/Binding.csproj --region MultipleArgs
var command = new RootCommand
{
    new Option("--a-string") { Argument = new Argument<string>() },
    new Option("--an-int") { Argument = new Argument<int>() }
};

command.Handler = CommandHandler.Create(
    (string aString, int anInt) =>
    {
        Console.WriteLine($"{aString}");
        Console.WriteLine($"{anInt}");
    });

await command.InvokeAsync("--an-int 123 --a-string \"Hello world!\" ");
```
````

Reply:
@jonsequitur, you have a valid point giving that the sample will include/focus on C# code.
However a one could argue that this code block doesn't have any C# code _yet!_.

Reply:
It does if you run it with `dotnet-try`, due to the code fence annotations.